release-notes/mac/release-notes.md

```diff
@@ -1,5 +1,27 @@
-# v1.1.245 (Feb 6 - 2024)
+# v1.3.253 (Feb 22 - 2024)
+## Bug Fixes/Improvements
```

Comment:
Missing features. 

Focus Alignment Score + Choosing which habits to do from a routine are new features.

A lot of this is really low level and not relevant for outside users.